### ISSUE TITLE
Tuning from rqt_reconfigure

### ISSUE THREAD

**Issue by superjax** [BYU-MAGICC/fcu_io2#4]
Hey, I've started working on the controller.  I have some philosophical questions though about how to deal with tuning, and everything else.

Should we be able to tune things from rqt_reconfigure?  I'm asking, because I want to know where we expect the user to interact with fcu_io.  I think we should probably support rqt_reconfigure and other ros-based tools before the web app.  What do you think @dpkoch @gellings?

**Comment by superjax** [comment]
I started on the `dynamic_reconfigure` stuff, but I am realizing that I am a little over my head with @dpkoch's snazzy parameter manager.  I'm going to leave it be, since it's pretty low priority right now, but when you get back, you'll have to walk me through how the param server works so I can finish hooking up the ROS parameter server.

**Comment by dpkoch** [comment]
So I think we've decided we're not using the ROS parameter server, which means we're not using dynamic reconfigure? Also I think we discussed that we don't want to maintain two parameter lists (one in the firmware, one for the dynamic reconfigure configuration)?

So should we say the answer is no and close this issue? Or do we still want to figure out some way to do this? Another option would be to have a tuning tab in the configurator that lets you easily set the relevant parameters using the ROS service API we already have. Thoughts?
